classes/MatricesProcessor.ts

```diff
@@ -8,4 +8,18 @@ export class MatricesProcessor extends MathProcessor {
     this.parsedMatrix = matrix(matrixExpresssion);
     return det(this.parsedMatrix);
   }
+
+  public matrixMultiplication(matrix1: number[][], matrix2: number[][]) {
+    const size1 = size(matrix1);
+    const size2 = size(matrix2);
+
+    if (size1[1] !== size2[0]) {
+      throw new Error("Inner dimensions do not match");
+    }
+
+    const parsedMatrix1 = matrix(matrix1);
+    const parsedMatrix2 = matrix(matrix2);
+
+    return multiply(parsedMatrix1, parsedMatrix2);
```

Comment:
Looks good, we should be returning plain javascript types though. Make this a `number [][]` type?